ci/ciimage/opensuse/install.sh

```diff
@@ -18,6 +18,7 @@ pkgs=(
   boost-devel libboost_date_time-devel libboost_filesystem-devel libboost_locale-devel libboost_system-devel
   libboost_test-devel libboost_log-devel libboost_regex-devel
   libboost_python3-devel libboost_regex-devel
+  perl-TimeDate # drop this once obs#1187867 made it into factory
```

Comment:
Since you're not trying to update the opensuse image today, only the Arch one, I would suggest letting opensuse fix this bug and having meson pick that fix up next week on the weekly refresh job.

You're not going to fix the opensuse image anyway unless you get dub to work, but that's tracked by #12143

Reply:
Thanks for debugging the upstream opensuse issue and submitting a fix to them, though. :)

Reply:
coincidentally im probably the one at fault, given that I submitted lcov 2.0 in the first place 🙃